services/api_service.ts

```diff
@@ -22,7 +22,7 @@ class APIImpl implements APIService {
     this.COHERE_API_KEY = key;
 
     if (version === undefined) {
-      this.COHERE_VERSION = "2021-11-08"; // currently latest, update when we version better
+      this.COHERE_VERSION = "2022-12-06"; // currently latest, update when we version better
```

Comment:
out of PR scope, but just sharing what comes to mind: 
not sure why we use this instead of the package version? is it for the model version?
regardless, I think it makes more sense to use it as a default variable instead of hard coding (either in env, or we can even define it directly in `package.json` along the version)

Reply:
I agree that this is worth revisiting. the idea was that we would let the users pick between different API versions, but since the library would break for older versions anyway, this is not helpful.

Reply:
it shouldn't break (using the same format).
I'm just saying that the default version (date) can be specified as some sort of config (possibly in the package.json) rather than hard coding inside the code for quicker access

Reply:
API versions bumps introduce breaking changes, and since our SDKs don't maintain support for old versions, things would start breaking. Anyway, going to resolve this, but we can refactor in another PR if you feel like it's an improvement!